modules/govuk_jenkins/files/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy

```diff
@@ -336,10 +336,12 @@ def publishGem(String repository, String branch) {
     sh("gem push ${repository}-${version}.gem")
   }
 
-  def taggedReleaseExists = sh(
-    script: "git ls-remote --exit-code --tags origin v${version}",
-    returnStatus: true
-  ) == 0
+  sshagent(['govuk-ci-ssh-key']) {
+    def taggedReleaseExists = sh(
```

Comment:
I think you would now need to declare taggedReleaseExists before the closure passed to sshagent, but I wasn't able to verify this locally.